db/warehouse/migrate/20240218222444_move_existing_overrides.rb

```diff
@@ -0,0 +1,78 @@
+class MoveExistingOverrides < ActiveRecord::Migration[6.1]
+  def up
+    p_t = GrdaWarehouse::Hud::Project.arel_table
+    {
+      act_as_project_type: :ProjectType,
+      # hud_continuum_funded: :ContinuumProject,
+      housing_type_override: :HousingType,
+      operating_start_date_override: :OperatingStartDate,
+      operating_end_date_override: :OperatingEndDate,
+      hmis_participating_project_override: :HMISParticipatingProject,
+      target_population_override: :TargetPopulation,
+    }.each do |override, source|
+      GrdaWarehouse::Hud::Project.where.not(override => nil).each do |project|
+        HmisCsvImporter::ImportOverride.where(
+          file_name: 'Project.csv',
+          matched_hud_key: project.ProjectID,
+          replaces_column: source,
+          replacement_value: project[override],
+          data_source_id: project.data_source_id,
+        ).first_or_create
```

Comment:
how many overrides will need to be created? Would it be worth it to collect them and bulk insert to reduce the runtime for this migration?